cityofaustin/atd-moped
# Edit signal components

## Associated issues
https://github.com/cityofaustin/atd-data-tech/issues/11114

This updates the edit component attributes form to be able to update the signal, subcomponents, and description of an existing signal component.

## Testing
**URL to test:** 

local or https://md-11114-edit-signal-component--atd-moped-main.netlify.app/moped/

**Steps to test:**
1. Create a new component and select either **Signal - PHB** for **Signal - Traffic**
2. Select a signal, add one or more subcomponents, and add a description
3. Click **Save**
4. Click your new component's list item on the left and then click **Edit**
5. When the edit modal appears, click **Attributes**
6. Update the signal, subcomponents, and description
7. Try updating each of those individually
8. Try updating different combinations
9. You should see your component update each time

---
#### Ship list
